sublime-package.json

```diff
@@ -0,0 +1,738 @@
+                        }
+                      },
+                      "default": {
+                        "summary": "$1",
+                        "tags": [
+                          "@var ${1:$SYMBOL_TYPE}"
+                        ]
+                      },
+                      "description": "An object that describes the format of generated property phpdoc. The following snippet variables are available: SYMBOL_NAME; SYMBOL_KIND; SYMBOL_TYPE; SYMBOL_NAMESPACE.",
+                      "scope": "window"
+                    },
+                    "intelephense.phpdoc.functionTemplate": {
+                      "type": "object",
+                      "properties": {
+                        "summary": {
+                          "type": "string",
+                          "description": "A snippet string representing a phpdoc summary."
+                        },
+                        "description": {
+                          "type": "string",
+                          "description": "A snippet string representing a phpdoc description."
+                        },
+                        "tags": {
+                          "type": "array",
+                          "items": {
+                            "type": "string"
+                          },
+                          "description": "An array of snippet strings representing phpdoc tags."
+                        }
+                      },
+                      "default": {
+                        "summary": "$1",
+                        "tags": [
+                          "@param ${1:$SYMBOL_TYPE} $SYMBOL_NAME $2",
+                          "@return ${1:$SYMBOL_TYPE} $2",
+                          "@throws ${1:$SYMBOL_TYPE} $2"
+                        ]
+                      },
+                      "description": "An object that describes the format of generated function/method phpdoc. The following snippet variables are available: SYMBOL_NAME; SYMBOL_KIND; SYMBOL_TYPE; SYMBOL_NAMESPACE.",
+                      "scope": "window"
+                    },
+                    "intelephense.phpdoc.useFullyQualifiedNames": {
+                      "type": "boolean",
+                      "default": false,
+                      "description": "Fully qualified names will be used for types when true. When false short type names will be used and imported where appropriate. Overrides intelephense.completion.insertUseDeclaration.",
+                      "scope": "window"
+                    },
+                    "intelephense.trace.server": {
+                      "type": "string",
+                      "enum": [
+                        "off",
+                        "messages",
+                        "verbose"
+                      ],
+                      "default": "off",
+                      "description": "Traces the communication between VSCode and the intelephense language server.",
```

Comment:
This mentions VSCode and also probably the setting doesn't work for us anyway.

Reply:
I try to set it to `"verbose"` but there seems no extra message sent from it. I guess, yes, we can just remove it. :+1: